.github/workflows/ci.yaml

```diff
@@ -48,7 +48,7 @@ jobs:
         run: |
           mvn -B -q install --file pom.xml -Drun311tests=${{ matrix.run311tests }} -Drun40tests=${{ matrix.run40tests }} -Drun41tests=${{ matrix.run41tests }}
   build-dse:
-    if: ${{ github.ref == 'refs/heads/master' && github.event_name == 'push'}}
```

Comment:
Would we maybe not want to combine the repo owner condition with the old conditions rather than replacing?

Reply:
Actually, the jobs run on `push` or `pull_request`, except this one would only run on `push`. So now it runs on either, so long as the owner is k8ssandra. Which makes it like the OSS jobs.